docarray/array/mixins/io/pushpull.py

```diff
@@ -250,6 +251,8 @@ def _get_chunk(_batch):
         if response.ok:
             return response.json()['data']
         else:
+            if response.status_code >= 400 and 'readableMessage' in response.json():
+                response.reason = response.json()['readableMessage']
```

Comment:
u should use HTTPStatus for somehing. There must be some constant from there, 400 seems magic to me

Reply:
I would've used it if it was an equal check (==).
I think it is more readable to use the actual number in comparison (it just means failed response).
The requests library does the same actually
![image](https://user-images.githubusercontent.com/15269265/196638019-bdd6fcde-4078-4a90-8845-aabb6fe8467b.png)